test/E2ETests/E2ETests/Helpers/TableHelpers.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Data.Tables;
+using Microsoft.Azure.Documents;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Azure.Functions.Tests.E2ETests;
+
+namespace Microsoft.Azure.Functions.Worker.E2ETests.Helpers
+{
+    public static class TableHelpers
```

Comment:
Make sure to include a Dispose method to dispose of the client when done

Reply:
I have a dispose method[ here](https://github.com/Azure/azure-functions-dotnet-worker/pull/1506/files/eda9939a70e1f0358eae19309ac72fcc25286867#diff-f119006f256a571974deb81d3bd8372fdb030f44357611442cdad9bcc29b8cd6R133). Does that work?